dev/AppLifecycle/ExtensionContract.h

```diff
@@ -39,7 +42,7 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
                 for (const auto& extension : c_extensionMap)
                 {
                     std::wstring contractId = pair.Value().c_str();
```

Comment:
>wstring [](http://example.com/codeflow?start=25&length=7)

No point using `std::wstring` if all we care about is the `c_str()`.

Also, the `pair` doesn't change through the loop, don't need to recalculate it each time. Can capture once up front.

```cpp
auto contractId = pair.Value();
for (const auto& extension : c_extensionMap)
{
    if (CompareStringOrdinal(contractId.c_str(), -1, extension.contractId, -1, TRUE) == CSTR_EQUAL) { ... }
}
```